src/functions/scrapeSubreddit.js

```diff
@@ -1,14 +1,14 @@
-const { Function } = require("klasa-functions");
-const { get } = require("snekfetch");
+const { Function } = require("@kcp/functions");
+const { util: { fetch } } = require("../index");
 
 module.exports = class extends Function {
 
-    async run(subreddit) {
+    run(subreddit) {
         subreddit = typeof subreddit === "string" && subreddit.length !== 0 ? subreddit : "puppies";
-        return get(`https://imgur.com/r/${subreddit}/hot.json`)
+        return fetch(`https://imgur.com/r/${subreddit}/hot.json`)
             .then(res => {
```

Comment:
Async/Await please

Reply:
It returns a promise which is why it's already awaited in where the function is being used.

Reply:
that makes no sense. Of course fetch is a promise but to make this code cleaner you should be using async/await please